tests/state_api_test.cpp

```diff
@@ -182,7 +182,7 @@ TEST_F(StateAPITest, DISABLED_dpos_integration) {
   // EXEC_AND_CHECK({});
 }
 
-TEST_F(StateAPITest, eth_mainnet_smoke) {
+TEST_F(StateAPITest, DISABLED_eth_mainnet_smoke) {
```

Comment:
hmm what changed that this test has to be disabled ?

Reply:
Not sure why I was disabling this, but I just forgot to remove this DISABLED_. It works fine